components/header/template.njk

```diff
@@ -3,20 +3,15 @@
   <div class="govuk-header__container govuk-width-container app-header__container">
     <div class="govuk-header__logo app-header__logo">
       <a href="{{ params.homepageUrl | default("/") | url }}" class="govuk-header__link govuk-header__link--homepage">
-        <span class="govuk-header__logotype">
-          {% if params.organisationLogo == "crown" %}
-            {% include "./_crown.svg" %}
-          {% elif params.organisationLogo == "royal-arms" %}
-            {% include "./_royal-arms.svg" %}
-          {% elif params.organisationLogo == "x-govuk" %}
-            {% include "./_x-govuk.svg" %}
-          {% elif params.organisationLogo %}
-            {{ params.organisationLogo | safe }}
-          {% endif %}
-          <span class="govuk-header__logotype-text">
-            {{ params.organisationName | default("GOV.UK") }}
-          </span>
-        </span>
+        {% if params.logotype == "x-govuk" %}
+          {% include "./_x-govuk.svg" %}
+        {% elif params.logotype.html %}
+          {{ params.logotype.html }}
+        {% elif params.logotype.text %}
+          <span class="govuk-header__logotype">{{ params.logotype.text }}</span>
```

Comment:
Should we use an `app-` prefix here maybe, as this class is no longer part of `govuk-frontend`?

```suggestion
          <span class="app-header__logotype">{{ params.logotype.text }}</span>
```

Reply:
Makes sense, changed.